tts_cli/tts_utils.py

```diff
@@ -138,7 +138,7 @@ def preprocess_dataframe(self, df):
         df['gender'] = df['DisplaySexID'].map(GENDER_DICT)
         df['voice_name'] = df['race'] + '-' + df['gender']
 
-        df['templateText_race_gender'] = df['text'] + df['race'] + df['gender']
+        df['templateText_race_gender'] = df['original_text'] + df['race'] + df['gender']
```

Comment:
Here I had to change the function to always use the English text so the hash values still matches.